common/src/Microsoft.Azure.IIoT.Abstractions/src/Serializers/Extensions/SerializerEx.cs

```diff
@@ -4,9 +4,9 @@
 // ------------------------------------------------------------
 
 namespace Microsoft.Azure.IIoT.Serializers {
-    using Microsoft.Azure.IIoT.Http;
+    using Microsoft.Azure.IIoT.Http;    
```

Comment:
Please remove trailing whitespaces introduced by this PR.

Reply:
Removing all trailing whitespaces from the solution, including in this PR. Subsequently, as agreed, raising a separate PR to include styling checks in editorconfig so code styling checks are part of the code for devs when they make changes in future.